samples/event_manager/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _event_manager:
+
+Event Manager
+#############
+
+The Event Manager sample demonstrates the functionality of the Event Manager library.
+It uses an event-driven architecture, where different modules communicate through by sending and processing events.
+
+
+Overview
+********
+
+The sample application consists of three modules that communicate using events:
+
+Sensor (``sensor_simulated.c``):
+  This module waits for a configuration event (which is sent by ``main.c``).
+  After receiving this event, it simulates measured data at constant intervals.
+  Every time the data is updated, the module sends the current values as measurement event.
+  When the module receives a control event from the Controller, it responds with an ACK event.
+
+Controller (``controller.c``):
+  This module waits for measurement events from the sensor.
+  Every time a measurement event is received, the module checks one of the measurement values that are transmitted as part of the event and, if the value exceeds a static threshold, sends a control event.
```

Comment:
I  removed the mention of value2 here - or does the threshold actually depend on that value?

Reply:
It checks if value2 exceeds static threshold - it is ok now